getting_started/workflow/best_practices/autoloads_versus_internal_nodes.rst

```diff
@@ -1,103 +1,109 @@
 .. _doc_autoloads_versus_internal_nodes:
 
-Autoloads versus internal nodes
+Autoloads versus regular nodes
+==============================
+
+Godot offers a feature to automatically load nodes at the root of your project,
+allowing you to access them globally, that can fulfill the role of a Singleton:
+:ref:`doc_singletons_autoload`. These auto-loaded nodes are not freed when you
+change the scene from code with ``SceneTree.change_scene``.
```

Comment:
Should we replace this with a direct link to the method in the API docs?

    change the scene from code with :ref:`SceneTree.change_scene <class_SceneTree_method_change_scene>`.